tests/integration/ha_tests/test_upgrade_from_stable.py

```diff
@@ -35,21 +35,24 @@
 @pytest.mark.abort_on_fail
 async def test_deploy_stable(ops_test: OpsTest) -> None:
     """Simple test to ensure that the PostgreSQL and application charms get deployed."""
-    await asyncio.gather(
-        ops_test.model.deploy(
-            DATABASE_APP_NAME,
-            num_units=3,
-            # TODO move to stable once we release
-            channel="16/beta",
-            trust=True,
-            base=CHARM_BASE_NOBLE,
-        ),
-        ops_test.model.deploy(
-            APPLICATION_NAME,
-            num_units=1,
-            channel="latest/edge",
-            base=CHARM_BASE,
-        ),
+    # TODO: rollback to ops_test.model.deploy syntax when we release to stable.
+    await ops_test.juju(
```

Comment:
This will be rolled back as soon as a revision is published to the `16/beta` channel.